compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.w3c.kt

```diff
@@ -510,22 +510,22 @@ internal class ComposeWindow(
          * - changedTouches contains only a Touch of a changed pointer, but compose needs all pointers,
          *   therefore we take targetTouches in this case;
          */
-        val touches = if (event.targetTouches.length > event.changedTouches.length) {
-            event.targetTouches.asList()
+        val isChangedTouchPressed =
+            eventType == PointerEventType.Press || eventType == PointerEventType.Move
+        val touches = if (isChangedTouchPressed) {
+            event.targetTouches.asList().fastMap { it to true }
         } else {
-            event.changedTouches.asList()
+            event.changedTouches.asList().fastMap { it to false } + event.targetTouches.asList()
+                .fastMap { it to true }
```

Comment:
As I see the final `touches` list always contains `event.targetTouches`.

In this case, wdyt about changing this to:

```kotlin
val touches = event.targetTouches.asList().fastMap { it to true }.toMutableList()
if (eventType == PointerEventType.Release) {
    touches2.addAll(event.changedTouches.asList().fastMap { it to false })
}
```